cds-snc/report-a-cybercrime
# Email victim

Fixes #1341 

# Description

if the user provides an email address, email them their report ID.

To test: 
1) get the values of the Notify environment variables from me and add to your `.env`
2) `npm run build && node server.js`
3) go to `localhost:3000`, fill out a report (including a correct contact email address) and submit.
4) You should get an email that includes your report ID.

# Any new packages installed?

notifications-node-client: library for Notify

# Checklist:


# Required followup work

After #1334 is addressed we can have an English template and a French template, and use the one corresponding to the language of the user. In this PR we just send one bilingual email.
